src/main/java/jalau/at18/kingoftokyo/DiceRoller.java

```diff
@@ -0,0 +1,39 @@
+package jalau.at18.kingoftokyo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DiceRoller {
+    private static final int COUNT_DICE = 6;
+    private RandomDice randomDice;
+
+    public DiceRoller() {
+        this.randomDice = new RandomDice();
+    }
+
+    //This method generated the list with random values.
+    public List<Dice> randomDice() { // No olvidar que aqui recibira la cantidad de randoms a generar a la lista
+        List<Dice> newlistDice = new ArrayList<>();
+        for (int count = 0; count < COUNT_DICE; count++) {
+            Dice dice = randomDice.generatedRandom();
+            newlistDice.add(dice);
+        }
+        return newlistDice;
+    }
+
+    public String showListDices() {
+        List<Dice> listDiceRandom = randomDice();
+        String numbersRandom = "";
+        for (int count = 0; count < listDiceRandom.size(); count++) {
+            numbersRandom = numbersRandom + String.valueOf(listDiceRandom.get(count).getDiceFace().getRepresentativeValue());
+        }
+        return numbersRandom;
+    }
+
+    public boolean verifyShowListDices() {
```

Comment:
Should this method actually be a unit test?

Reply:
At the moment it is only to verify that the showListDices() method is returning a value and is not empty, it will be removed in a future commit.

Reply:
ok